content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -71,35 +85,18 @@ This analysis uses the same logic and error messages as when using `istioctl ana
 
 For example. if you have a misconfigured gateway on your "ratings" virtual service, running `kubectl get virtualservice ratings` would give you something like:
 
-{{< text yaml >}}
-apiVersion: networking.istio.io/v1alpha3
+{{< text syntax=yaml snip_id=vs_yaml_with_status >}}
+apiVersion: networking.istio.io/v1beta1
 kind: VirtualService
-metadata:
-  annotations:
-    kubectl.kubernetes.io/last-applied-configuration: |
-      {"apiVersion":"networking.istio.io/v1alpha3","kind":"VirtualService","metadata":{"annotations":{},"name":"ratings","namespace":"default"},"spec":{"hosts":["ratings"],"http":[{"route":[{"destination":{"host":"ratings","subset":"v1"}}]}]}}
-  creationTimestamp: "2019-09-04T17:31:46Z"
-  generation: 11
-  name: ratings
-  namespace: default
-  resourceVersion: "12760039"
-  selfLink: /apis/networking.istio.io/v1alpha3/namespaces/default/virtualservices/ratings
-  uid: dec86702-cf39-11e9-b803-42010a8a014a
-spec:
-  gateways:
-  - bogus-gateway
-  hosts:
-  - ratings
-  http:
-  - route:
-    - destination:
-        host: ratings
-        subset: v1
+...
 status:
   validationMessages:
+...
```

Comment:
Is this message coming out in the middle between other messages when you run the test? What other messages? If you can configure the test environment to only produce the one error, or to make it first or last, you could simplify the doc by removing one or both `...` around the messages.